### PR TITLE
Adding bio for Nicole Witek

### PR DESCRIPTION
**Pull request by nameche**
This pull request addresses issue #13 
- [x] Favorite food?

### CODE REVIEW

**Comment by nameche** [comment]
@lcampos71 please see my pull request. Thanks!

File: Nicole-bio.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## Nicole's Bio ##
```
Comment (nameche) [inline Nicole-bio.md]: @CoderRyan800 please see my bio

Comment (CoderRyan800) [inline Nicole-bio.md]: Hi Nicole,

Looks good!  Do you want to add one or two hobbies?

Ryan

Comment (CoderRyan800) [inline Nicole-bio.md]: Looks great!

**Pull request merged**